### ISSUE TITLE
can't install on mac

### ISSUE THREAD

**Issue by RRickyHunter** [anatosun/plex2mix#13]
Hi there.
I'm not a programmer, so please forgive me if I don't understand your answer :)

I followed your instructions. But before I had to install python, and before that commandline tools and before that homebrew... eventually I could use the cited command. This was the result:
python3 setup.py install --user
Traceback (most recent call last):
  File "/Users/ricky/Downloads/plex2mix-main/setup.py", line 1, in <module>
    from setuptools import setup, find_packages
ModuleNotFoundError: No module named 'setuptools'

then I tried with this:
plex2mix-main % sudo pip3 setup.py install --user
ERROR: unknown command "setup.py"

As far as I'm understanding It can't find setuptool, so I tried:
brew setuptools
and again "Error: Unknown command: setuptools

I started the terminal in the correct folder of plex2mix.
I'm not using bash but zsh, I'm running last mac osx Sonoma 14.2.1 and I'm on a MacBook Pro Max M2 2023.

If you need other info, please let me know.
Cheers.

**Comment by bencorryp** [comment]
You need to run a pip install for setuptools.
